core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -88,6 +88,13 @@ object Defaults {
   val LogCleanerMinCleanRatio = 0.5d
   val LogCleanerEnable = true
   val LogCleanerDeleteRetentionMs = 24 * 60 * 60 * 1000L
+  val MD2 = "MD2"
```

Comment:
There are three configs right now that have a list for valid values:  LogCleanupPolicy, LogMessageTimestampType and SslClientAuth.

LogCleanupPolicy and SslClientAuth use constants defined in Defaults object. LogMessageTimestampType define allowed values directly without using constants. I followed same approach used by the majority :). I prefer to use the constants otherwise we will have to duplicate the default value.

.define(LogCleanupPolicyProp, STRING, Defaults.LogCleanupPolicy, in(Defaults.Compact, Defaults.Delete), MEDIUM, LogCleanupPolicyDoc)

.define(SslClientAuthProp, STRING, Defaults.SslClientAuth, in(Defaults.SslClientAuthRequired, Defaults.SslClientAuthRequested, Defaults.SslClientAuthNone), MEDIUM, SslClientAuthDoc)

.define(LogMessageTimestampTypeProp, STRING, Defaults.LogMessageTimestampType, in("CreateTime", "LogAppendTime"), MEDIUM, LogMessageTimestampTypeDoc)